guides/source/upgrading_ruby_on_rails.md

```diff
@@ -180,6 +180,26 @@ Technically, however, controllers could configure `helpers_path` to point to a d
 
 The default HTTP status code used in `ActionDispatch::SSL` when redirecting non-GET/HEAD requests from HTTP to HTTPS has been changed to `308` as defined in https://tools.ietf.org/html/rfc7538.
 
+### Active Storage now requires Image Processing
+
+When processing variants in Active Storage, it's now required to have [image_processing gem](https://github.com/janko-m/image_processing) bundled instead of directly using `mini_magick`. Image Processing is configured by default to use `mini_magick` behind the scenes, so the easiest way to upgrade is just replacing the `mini_magick` gem for `image_processing` gem and making sure to remove the explicit usage of `combine_options` since it's no longer needed.
```

Comment:
Let's leave out the word "the" for the reasons mentioned in https://github.com/rails/rails/pull/37946. 

There were also a couple missing articles.

```suggestion
When processing variants in Active Storage, it's now required to have the [image_processing gem](https://github.com/janko-m/image_processing) bundled instead of directly using `mini_magick`. Image Processing is configured by default to use `mini_magick` behind the scenes, so the easiest way to upgrade is by replacing the `mini_magick` gem for the `image_processing` gem and making sure to remove the explicit usage of `combine_options` since it's no longer needed.
```